cmd/ps.go

```diff
@@ -112,45 +114,80 @@ func parseEntry(line string) (psresult, error) {
 	return psresult{pid: pid, active: activeTime, command: command}, nil
 }
 
-func parseLines(node string, chunk string) []psresult {
+func parseStatLines(chunk string) ([]psresult, error) {
```

Comment:
Might help to re-arrange some of these functions as something like

```
// PS POSIX
<posix stuff>

// PS POWERSHELL
<powershell stuff>
```

so a maintainer can immediately see what's what.